src/ws/cockpitbranding.c

```diff
@@ -22,11 +22,36 @@
 #include "common/cockpitwebserver.h"
 #include "common/cockpittransport.h"
 #include "common/cockpitsystem.h"
+#include "common/cockpitconf.h"
 
 #include "cockpitauth.h"
 #include "cockpitbranding.h"
 #include "cockpitwebservice.h"
 
+static gboolean
+add_config_branding_dirs (GPtrArray *dirs)
+{
+  const gchar * const* config_dirs;
+  gboolean found_any = FALSE;
+
+  config_dirs = cockpit_conf_get_dirs ();
+  while (config_dirs && config_dirs[0])
+    {
+      g_autofree gchar *config_path = g_build_filename (config_dirs[0], "cockpit", "branding", NULL);
+      g_autofree gchar *config_css = g_build_filename (config_path, "branding.css", NULL);
```

Comment:
Why do we check for this specific file instead of just the `config_path` directory?  The commit message and code comments don't mention this, right?

Reply:
We need to document this in `doc/branding.md`.